hnn_core/dipole.py

```diff
@@ -69,7 +68,7 @@ def simulate_dipole(net, n_trials=None, record_vsoma=False,
         raise TypeError("record_isoma must be bool, got %s"
                         % type(record_isoma).__name__)
 
-    dpls = _BACKEND.simulate(net, postproc)
+    dpls = _BACKEND.simulate(net, n_trials, postproc)
```

Comment:
If `n_trials` is updated in the `net.params`. why does it need to be passed explicitly?

Reply:
This is to avoid having to pass the updated dict around! There's only one place in `mpi_child` that prevents getting rid of L56 here. I'll take a look once #211 is in.

This comes down to my preference of thinking about the network as the input to potentially multiple different simulations: `n_trials` is not intuitively a property of the network, right?